b2sdk/_botocore/awsrequest.py

```diff
@@ -82,6 +83,11 @@ def request(self, method, url, body=None, headers=None, *args, **kwargs):
         self._response_received = False
         if headers.get('Expect', b'') in [b'100-continue', '100-continue']:
             self._expect_header_set = True
+            timeout = headers.pop('X-Expect-100-Continue-Timeout-Seconds', self._continue_timeout)
```

Comment:
nitpick: drop `-Seconds` part of the name; AFAIK there are already timeout settigns http headers, none of which adds `-seconds` (Cache-Control, Keep Alive etc)